src/main/java/com/facebook/openwifirrm/modules/ProvMonitor.java

```diff
@@ -160,8 +194,79 @@ protected void syncDataToProv(
 			topo.values().stream().mapToInt(x -> x.size()).sum()
 		);
 
+		// Sync zone configs
+		deviceDataManager.updateZoneConfig(
+			configMap -> {
+				for (
+					Map.Entry<String, RRMDetails> entry : rrmDetails.entrySet()
+				) {
+					String venue = entry.getKey();
+					if (venue.isEmpty()) {
+						logger.error("Venue is blank for an RRM enabled zone");
+						continue;
+					}
+
+					String zone = venueIdToName.get(venue);
+					if (zone == null) {
+						logger.error(
+							"Venue name {} is not found in id mapping",
+							venue
+						);
+						continue;
+					}
+
+					DeviceConfig cfg = configMap
+						.computeIfAbsent(zone, k -> new DeviceConfig());
+					cfg.enableRRM =
+						cfg.enableConfig = cfg.enableWifiScan = true;
```

Comment:
you don't need to do this here because the block below syncing device configs will take care of it -- it shouldn't be in two places